src/dispatch/incident/flows.py

```diff
@@ -310,7 +310,6 @@ def incident_create_resources_flow(
     return incident_create_resources(incident=incident, db_session=db_session)
 
 
-@background_task
```

Comment:
Encountered race condition where `commit()` was called before the `incident` object existed in the database, `None` is called on `commit()`